project-communications/newsletter-20200423.md

```diff
@@ -0,0 +1,202 @@
+
+### Workshop
+
+Wireframing workshop: On the 27th March we ran a wireframing workshop over Zoom. 
+The workshop was a mix of people from Autistica, Fujitsu, the Turing, and community members. 
+You can see the notes from the meeting by following this link: https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-wireframing-workshop/project-management/workshops/wireframing-March2020-notes.md
+
+# Upcoming opportunities
+
+### User testing
+
+We've started running user testing sessions with the Fujitsu team, and we're looking for people to help give feedback. 
+
+These will take place remotely by videocall, and you will be paid in line with involve guidelines for you time. 
+
+Please look out for an email from Autistica! 
+
+You can get in touch with Georgia at [gaitkenhead@turing.ac.uk](mailto:gaitkenhead@turing.ac.uk) if you'd like to be involved  or if you would prefer any adjustments to be made. 
+
+We will only be looking for one person for each session, but the sessions will reccur regularly while the platform is being built.
+
+### Social Meet-ups 
+
+A few of you have said that you'd like to connect with people involved in the project in a social setting and would be interested in getting to know each other better and meeting new people! :smiley:. 
+One way to do it would be to have a time slot for people to join a videocall and chat if they want to. 
+
+It would be great to hear from you about:
+* whether you would be interested in having a regular time slot for an online social meeting
+* what activities might be fun to do? Some thoughts from the community so far are:
+    * an open discussion
+    * a quiz 
+    * a reading group
+* when would be a good time for you? Any times you definitely can't do? 
+* The Turing can host a videocall on Zoom, how would this suit you? 
+
+Please get in touch with Georgia at [gaitkenhead@turing.ac.uk](mailto:gaitkenhead@turing.ac.uk) if you'd like to give your thoughts or be involved! 
+
+# Introducing Otis
+
+Every month, we've decided to introduce a new member of the community to show the range of people involved in the project and the variety of contributions they're making.
+If you'd like to be interviewed, or to interview someone else, please get in touch with Georgia at gaitkenhead@turing.ac.uk
+
+This month, we're delighted to introduce Otis Smith. 
+
+Otis has been involved in multiple aspects of the project, including giving feedback on designs, presentations, and agendas; helping to design the labelling system for the focus group summaries; helping to prioritise tasks; giving ideas and insights in workshops; and co-authoring the blog written for World Autism Awareness day. 
+He also co-wrote this newsletter!  
+
+Here he is:
+
+![](https://i.imgur.com/c7d8kQ7.jpg)
```

Comment:
Can you add this picture to the repo please and link.

Reply:
Added to the repo here: https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-communications/images